src/Perlang.Tests/ConsoleApp/ProgramTest.cs

```diff
@@ -81,7 +82,7 @@ public void can_call_function_from_expression()
                 subject.Run("fun hello(): void { print 1; }", FatalWarningsHandler);
                 subject.Run("hello()", FatalWarningsHandler);
 
-                Assert.Equal(new List<string> { "1" }, output);
+                output.Should().Equal("1");
```

Comment:
For cases like these, Fluent Assertions feels quite obviously like a "better" approach. For others, maybe not so much. Anyway, I think it does make sense in another way as well in that it makes us less dependent on xUnit. If we ever want to switch to some other unit testing framework, we are much easier off if we're already using Fluent Assertions for our assertions => gain.